Makefile

```
@@ -37,6 +37,7 @@ migrate:
 	@NODE_ENV=development knex migrate:latest --cwd ../api/src/sql/ --migrations-directory ./migrations
 migrate-down:
 	@NODE_ENV=development knex migrate:rollback --all --cwd ../api/src/sql/ --migrations-directory ./migrations
+remigrate: migrate-down migrate
```

I would add docs on this.
```suggestion
remigrate: ## Refreshes the database schema and clears all data.
         migrate-down migrate
```

---

I'm not sure if that is what this does though, and/or if it needs to be run each time inframock is started.

@cosa65 I need helpz